CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for stripes-connect
 
+## 5.7.0 (IN PROGRESS)
+
+* Use `UNSAFE_componentWillReceiveProps` for a quieter console. Refs STCON-70.
```

Comment:
I see changes only in this file. Should it be some additional code changes?

Reply:
LOL!

Reply:
Yeah, well, the changes are _super quiet_.

🤦